<ui version="4.0" >
 <class>ChannelPlotWidget</class>
 <widget class="QWidget" name="ChannelPlotWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>130</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>ChannelPlotWidget</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="SideCar::GUI::HealthAndStatus::SamplePlotWidget" native="1" name="plot_" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
       <horstretch>100</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <layout class="QVBoxLayout" >
     <property name="leftMargin" >
      <number>2</number>
     </property>
     <property name="rightMargin" >
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="max_" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font" >
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="expected_" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font" >
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>12345</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="min_" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font" >
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <property name="leftMargin" >
      <number>6</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="channelName_" >
       <property name="font" >
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>NegativeVideo</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="sampleLabel_" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font" >
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Delta Value:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sample_" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font" >
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="dropsLabel_" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font" >
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Drops:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="drops_" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font" >
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SideCar::GUI::HealthAndStatus::SamplePlotWidget</class>
   <extends>QWidget</extends>
   <header>SamplePlotWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../guiIcons.qrc" />
 </resources>
 <connections/>
</ui>
